src/code42cli/cmds/securitydata.py

```diff
@@ -35,8 +39,55 @@
 
 logger = get_main_cli_logger()
 MAX_EVENT_PAGE_SIZE = 10000
-
 SECURITY_DATA_KEYWORD = "file events"
+
+
+def exposure_type_callback():
+    def callback(ctx, param, arg):
+        if arg:
+            if ctx.obj.profile.use_v2_file_events == "True":
+                raise Code42CLIError(
+                    "Exposure type (--type/-t) filter is incompatible with V2 file events. Use the event action (--event-action) filter instead."
+                )
+            ctx.obj.search_filters.append(ExposureType.is_in(arg))
+        return arg
+
+    return callback
+
+
+def event_action_callback():
+    def callback(ctx, param, arg):
+        if arg:
+            if ctx.obj.profile.use_v2_file_events == "False":
+                raise Code42CLIError(
+                    "Event action (--event-action) filter is incompatible with V1 file events.  Upgrade your profile to use the V2 file event data model with `code42 profile update --use-v2-file-events`"
+                )
+            ctx.obj.search_filters.append(v2_filters.event.Action.is_in(arg))
+        return arg
+
+    return callback
+
+
+def get_all_events_callback():
+    def callback(ctx, param, arg):
+        if not arg:
+            if ctx.obj.profile.use_v2_file_events == "True":
+                risk_values = [
```

Comment:
This is missing `NO_RISK_INDICATED`. 

We could also just do `v2_filters.risk.Severity.exists()` and that future-proofs this in the event that we add any new severity levels.

Reply:
Ah maybe I misunderstood this filter? I'm under the impression that if `--include-non-exposure` is _not_ called, then this callback happens.  In the V1 case we make sure there's an exposure type, and in the V2 case I think the equivalent would be making sure there's some level of severity attached.  For sure let me know if I'm wrong on this though, was a little unclear on it

Reply:
Nope you're right, I forgot how that worked. This is correct.

Reply:
Though we could do `v2_filters.risk.Severity.not_eq(v2_filters.risk.Severity.NO_RISK_INDICATED)` to handle any future severity changes.

Reply:
Oh thats a good call, I like it